com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/archive/ManifestBuilder.java

```diff
@@ -454,35 +458,29 @@ public void addResourceEntry(String url, byte[] data, int flags) throws IOExcept
            and thus create a partial archive that has to be updated (through
            LiveUpdate) before that CollectionProxy can be loaded.
         */
-        ResourceNode candidate = null;
+
+        List<String> dependants = new ArrayList<String>();
+
+        ResourceNode candidate = pathToNode.get(filepath);
```

Comment:
Don't traverse the entire tree to find the node.
Saved `68` seconds(!) in the test project.